src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -112,6 +112,7 @@
 		mutableAudioOrVideoContainers.put(FormatConfiguration.THREEGPP2, new AudioVariantInfo(new THREEG2A(), FormatConfiguration.THREEGA));
 		// XXX WEBM Audio is NOT MKA, but it will have to stay this way until WEBM Audio is implemented.
 		mutableAudioOrVideoContainers.put(FormatConfiguration.WEBM, new AudioVariantInfo(new MKA(), FormatConfiguration.WEBA));
+		mutableAudioOrVideoContainers.put(FormatConfiguration.WMV, new AudioVariantInfo(new WMA(), FormatConfiguration.WMA));
```

Comment:
@Sami32 I haven't seen how you fixed it, so I don't know, but I disagree that this is "a quick patch". This was something that was forgotten in #1285 and should have been a part of that PR. The ASF problem is "a separate issue" as far as I can see. The "confusion" is that WM use a previous version of ASF as its container structure, so in a sense a WM file is also an ASF file. The opposite isn't true though. If we don't throw this confusion into mix, the above is, as far as I can see, correct. UMS parse WM as WMV "by default", and if the initial parsing of the format is correct (that is, doesn't parse ASF as WM), then this should work fine.

The ASF problem should be addressed, but my intention here was just to fix what was forgotten in #1285.